test/test.js

```diff
@@ -25,11 +27,27 @@ var qunitRunner = new JsReporters.QUnitAdapter(QUnit)
 var qunitTestReporter = new JsReporters.TestReporter(qunitRunner, referenceData.QUnit)
 
 QUnit.done(function () {
-  process.exit(qunitTestReporter.ok ? 0 : 1)
+  if (!qunitTestReporter.ok) {
+    process.exit(1)
+  }
 })
 
 QUnit.config.autorun = false
 
 require('./qunit/tests.js')
 
 QUnit.load()
+
+var mocha = new Mocha()
+var mochaRunner = new JsReporters.MochaAdapter(mocha)
+var mochaTestReporter = new JsReporters.TestReporter(mochaRunner, referenceData.Mocha)
+
+mocha.addFile(
+  path.join(__dirname, 'mocha/tests.js')
```

Comment:
Why is this `addFile` call using `path.join`, but the other one (above) is just adding `test/mocha/tests.js`?